### PR TITLE
fix(k8s-eks): delete K8S LB service if exists recreating scylla nodes

### PR DESCRIPTION
**Pull request by vponomaryov**
Right now, if we try to recreate K8S scylla node we fail on the attempt to create already existing K8S LB service which just doesn't get deleted when it's target node/pod gets deleted.
So, consider this situation and delete such services if exist prior to creation of new one.

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

File: sdcm/utils/k8s.py [hunk 1]
```diff
@@ -166,6 +173,34 @@ def deploy(self):
...
 
+    def is_deleted(self):
+        try:
+            self.service
+        except RuntimeError:
+            return True
+        return False
+
+    def delete(self):
+        n, ns = self.name, self.namespace
```
Comment (bentsi) [inline sdcm/utils/k8s.py:184]: too short names...

File: sdcm/utils/k8s.py [hunk 2]
```diff
@@ -166,6 +173,34 @@ def deploy(self):
...
+
+        try:
+            LOGGER.debug(f"Trying to delete '{n}' K8S service in the '{ns}' namespace")
+            self.core_v1_api.delete_namespaced_service(
+                name=self.name, namespace=self.namespace, async_req=False)
```
Comment (bentsi) [inline sdcm/utils/k8s.py:189]: did you assign n, ns variable before try..except clause to use them here?

File: sdcm/utils/k8s.py (hunk 1 above)
Comment (vponomaryov) [reply sdcm/utils/k8s.py]: Done

File: sdcm/utils/k8s.py (hunk 2 above)
Comment (vponomaryov) [reply sdcm/utils/k8s.py:188]: Done

**Review by bentsi — APPROVED** [review]

**Pull request merged**